go.mod

```diff
@@ -22,22 +20,19 @@ require (
 	github.com/grpc-ecosystem/go-grpc-prometheus v1.2.0
 	github.com/grpc-ecosystem/grpc-gateway v1.14.3
 	github.com/jinzhu/gorm v1.9.12
-	github.com/jmespath/go-jmespath v0.3.0 // indirect
 	github.com/kelseyhightower/envconfig v1.4.0 // indirect
 	github.com/lib/pq v1.3.0
-	github.com/lyft/flyteidl v0.17.30
-	github.com/lyft/flytepropeller v0.2.13
-	github.com/lyft/flytestdlib v0.3.2
+	github.com/lyft/flyteidl v0.17.34
+	github.com/lyft/flytepropeller v0.2.59-0.20200612175034-00fd52a495b2
```

Comment:
is this a release version?

Reply:
nope... I'll cut a release version and update this PR when I merge Propeller changes... it's just for testing so far...